robot/Cumulus/resources/NPSP.py

```diff
@@ -635,19 +636,19 @@ def click_save(self, page):
             id="j_id0:theForm:j_id9:j_id10:saveBTN"
```

Comment:
Is this id a constant value?

Reply:
instead of using that long id with numbers, cant we use id contains saveBTN

Reply:
This code is preexisting in npsp.py not added by me and is not relevant to any of the tests in this PR. Related to GAU Robot tests.

Reply:
ok we can handle this as part of GAUs then